packages/grafana-ui/src/components/BigValue/PercentChange.tsx

```diff
@@ -8,8 +10,12 @@ export interface Props {
 }
 
 export const PercentChange = ({ percentChange, styles }: Props) => {
-  const percentChangeIcon =
-    percentChange && (percentChange > 0 ? 'arrow-up' : percentChange < 0 ? 'arrow-down' : undefined);
+  let percentChangeIcon: IconName | undefined = undefined;
```

Comment:
Good job with type protection here! This will ensure that a "0" icon never makes an appearance next to the actual percentChangeString.